### PR TITLE
fix layout

### PR DESCRIPTION
**Pull request by vipultiwari20**

### CODE REVIEW

File: crossword/css/crossword.css [hunk 1]
```diff
@@ -20,17 +20,47 @@
...
+				margin-left:0;
+				margin-right:0;
+				padding-left:0;
+				padding-right:0;
+			}
+			.clues2{
+				 float: right;
+				width: 150px;
+				font-size:18px;
+				t	ext-align:left;
```
Comment (dsandeep97) [inline crossword/css/crossword.css:51]: Fix this.

File: crossword/css/crossword.css [hunk 2]
```diff
@@ -20,17 +20,47 @@
...
+		width: 600px;
 		margin-left: auto;
 		margin-right:auto;
 		padding-left: 0;
 		padding-right:0;
 
 }
 
+@media screen and (max-width: 480px) {
+			.crossword {
```
Comment (dsandeep97) [inline crossword/css/crossword.css:32]: also indentation in this block

**Review by dsandeep97 — CHANGES_REQUESTED** [review]
some minor comments

File: crossword/css/crossword.css (hunk 1 above)
Comment (vipultiwari20) [reply crossword/css/crossword.css:51]: good catch !!

**Review by dsandeep97 — APPROVED** [review]

**Pull request merged**